templates/legal/terms-and-policies/cube-terms.html

```diff
@@ -45,14 +45,14 @@ <h2>2. Account</h2>
     <ul>
         <li>have a Service account;</li>
         <li>not use the Service for illegal or unauthorised purposes (or which encourage or permit illegal or authorised purposes), in infringement of a third party's’ rights or otherwise in accordance with these Terms of Service;</li>
```

Comment:
```suggestion
        <li>not use the Service for illegal or unauthorised purposes (or which encourage or permit illegal or authorised purposes), in infringement of a third party's rights or otherwise in accordance with these Terms of Service;</li>
```